release/rllib_tests/multi_node_checkpointing_compute_config.yaml

```diff
@@ -9,9 +9,9 @@ head_node_type:
 
 worker_node_types:
     - name: worker_node
-      instance_type: g3.8xlarge
-      min_workers: 1
-      max_workers: 1
+      instance_type: g3s.xlarge
+      min_workers: 2
+      max_workers: 2
```

Comment:
oh hmm if you change the number of workers here, you need to change the wait_for_node in release_tests.yaml as well

Reply:
oh shucks. It seems to have all worked out though?

Reply:
tests were previously passing.

Reply:
yes, in theory it will be a data race bug so it might or might not happen